nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -424,15 +521,16 @@ VelocitySmoother::dynamicParametersCallback(std::vector<rclcpp::Parameter> param
           shared_from_this(), odom_duration_, odom_topic_);
       }
     } else if (param_type == ParameterType::PARAMETER_DOUBLE_ARRAY) {
-      if (parameter.as_double_array().size() != 3) {
-        RCLCPP_WARN(get_logger(), "Invalid size of parameter %s. Must be size 3",
-            param_name.c_str());
+      size_t size = is_6dof_ ? 6 : 3;
+      if (parameter.as_double_array().size() != 3 && parameter.as_double_array().size() != size) {
```

Comment:
I don't understand the first check for `3`. Shouldn't the second be enough? It shouldn't be reconfigurable to a different size. It should always be 3 or always be 6, I think